action.yml

```diff
@@ -95,7 +95,7 @@ runs:
         HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: "1"
       run: |
         echo "::group::Installing QEMU, Docker client, and Docker Compose"
-        brew install docker docker-compose qemu 2>&1 | tee install.log
```

Comment:
@FHachez the build has failed because `--force` will only work for keg-only formulas and Python isn't one of them. The cli arg we need is `--overwrite`.